README.md

```diff
@@ -15,16 +19,19 @@ Essentially all workflows which take in chromosome-level files share filename re
+### Running on Terra (recommended)
+For Terra users, it is recommended to import via [Dockstore](https://dockstore.org/organizations/bdcatalyst/collections/UWGACAncestryRelatedness). Importing the correct JSON file for your workflow at the workflow field entry page will fill in test data and recommended runtime attributes for said test data. For example, load `vcf-to-gds-terra.json` for `vcf-to-gds.wdl`. If you are using your own data, please be sure to adjust your runtime attributes appropriately.  
 
-### Local users
-Cromwell does not manage resources well on local executions -- parameters such as `memory` and `disks` get ignored when Cromwell detects it is not running on the cloud. As a result, these pipelines (LD pruning especially) may get their processes killed by your OS for hogging too much memory, or completely lock up Docker, even on a relatively powerful machine running on downsampled test data. That being said, preliminary testing of these pipelines is performed on a local machine running OSX Catalina, so while we cannot officially support this method of execution, the only thing really blocking it from running smoothly on a local machine is Cromwell's resource management and the power needed by some of these algorithms. These issues can *generally* be avoided by changing the concurrent job limit in your Cromwell configuration. [See instructions here](https://docs.dockstore.org/en/develop/getting-started/getting-started-with-wdl.html#setting-up-the-dockstore-cli) for how to set it in the Dockstore CLI.
+### Running on your local machine
+Much preliminary testing and development of these pipelines was done by running Cromwell in "local mode," but we do not recommend this approach for doing actual analysis. Cromwell does not manage resources well on local executions. As a result, these pipelines (LD pruning especially) may get their processes killed by your OS and/or lock up Docker, even if running on downsampled data. These issues can *generally* be avoided by changing the concurrent job limit in your Cromwell configuration, and if you set this limit, you should find that all of the sample data in this repo should run on any Cromwell-and-Docker compatiable machine. [See instructions here](https://docs.dockstore.org/en/develop/getting-started/getting-started-with-wdl.html#setting-up-the-dockstore-cli) for how to set the concurrent job limit in the Dockstore CLI.
+
+### Running on an HPC
+These workflows have not been extensively tested in an HPC environment, but provided your HPC supports Cromwell and Docker, they should work as expected. You may which to run the checker workflows before doing actual analysis to ensure everything is running smoothly.
```

Comment:
"You may which to run the checker workflows before doing actual analysis to ensure everything is running smoothly."

Missing verb? Or something. This sentence feels off!

Reply:
Oops, that "which" was supposed to be "wish", thanks for spotting this!